### DISCUSSION TITLE
Any plan on lidar?

### DISCUSSION THREAD

**Discussion by hamiddashti** [eradiate/eradiate#357] [Feature requests]
Thanks for putting this together! Is there any plan to incorporate lidar into the model? 

Cheers

**Comment by leroyvn** [comment]
Hi @hamiddashti, we are currently focusing on passive remote sensing in the solar reflective domain (in situ or at the top of the atmosphere). LIDAR is on our roadmap, although with no foreseen deadline yet. We are aware of user interest, and we are considering starting work on it during Q4 2023 / Q1 2024 (some major changes to our scene API are required to improve user experience prior to that).

For our information, do you have a range of applications in mind?

**Reply by hamiddashti** [reply]
Hi @leroyvn . Sounds great and looking forward to it! It is really nice to see open-source, pythonic RTMs. Well, I personally use lidar for remote sensing of vegetation. We get the structure of the vegetation canopy from terrestrial and aerial laser scanners. It would be great, for example, to have vegetative scene point clouds and waveforms on top of the optical radiation budget. This helps us to establish links between lidar and optical data.  I hope it helps.
